src/shared/components/BaseToastMessages.tsx

```diff
@@ -0,0 +1,153 @@
+import {
+  Card,
+  Link,
+  CancelIcon,
+  CautionIcon,
+  CheckIcon,
+} from '@hypothesis/frontend-shared/lib/next';
+import classnames from 'classnames';
+
+import type { ToastMessage } from '../../sidebar/store/modules/toast-messages';
+
+type ToastMessageItemProps = {
+  message: ToastMessage;
+  onDismiss: (id: string) => void;
+};
+
+/**
+ * An individual toast message: a brief and transient success or error message.
+ * The message may be dismissed by clicking on it. `visuallyHidden` toast
+ * messages will not be visible but are still available to screen readers.
+ *
+ * Otherwise, the `toastMessenger` service handles removing messages after a
+ * certain amount of time.
```

Comment:
This sentence is no longer applicable, since this is a generic component which doesn't know anything about the sidebar store.